relay/api/schemas.py

```diff
@@ -17,12 +17,12 @@ class Meta:
     def _validate(self, data):
         nonce = data["nonce"]
         fees = data["fees"]
-        signature = data.get("signature", "")
+        signature = data["signature"]
         if not 0 <= nonce < 2 ** 256:
             raise ValidationError(f"nonce={nonce} is out of bounds")
         if not 0 <= fees < 2 ** 64:
             raise ValidationError(f"delegationFees={fees} is out of bounds")
-        if len(signature) != 65 and signature != "":
+        if len(signature) != 65 and len(signature) != 2:
```

Comment:
if len(signature) != 2 is the same as signature != "0x" we should use the latter. If it's something different, you should explain it.